src/engraving/rw/read400/tread.cpp

```diff
@@ -423,35 +423,6 @@ bool TRead::readItemProperties(EngravingItem* item, XmlReader& e, ReadContext& c
                 LOGW("EngravingItem::readProperties: could not link %s at staff %d", item->typeName(), mainLoc.staff() + 1);
             }
         }
-    } else if (tag == "lid") {
```

Comment:
Is this safe to remove? Were `<linked>` tags used in addition to this id in pre 301 scores? read400/tread is used in all older readers

Reply:
Yes, as far as all I can tell this "lid" (which I assume is short for linkID) tag only ever existed in 2.x versions and was only meant as a debugging tool. For all I can tell it had no functional use whatsoever